### ISSUE TITLE
[Feature] Voice implementation from SwiftDiscord?

### ISSUE THREAD

**Issue by jacksongoode** [SwiftcordApp/Swiftcord#83]
Just an idea for a feature/bootstrap, maybe voice implementation and other API's can be forked from https://github.com/nuclearace/SwiftDiscord.

**Comment by jacksongoode** [comment]
Actually, perhaps this would be a better maintained implementation https://github.com/SketchMaster2001/Swiftcord/pull/8.

**Comment by cryptoAlgorithm** [comment]
Oh wow, there are more `swift.*cord`s than I expect xD. I will take a look at them, and see if they are suitable to be adopted. Currently though, VC support is very far away on the timeline, so it would probably not be implemented so soon. Also, a better place to open issues would be in [DiscordKit](https://github.com/SwiftcordApp/DiscordKit), since thats the actual Discord API implementation.

**Comment by cryptoAlgorithm** [comment]
I'm keeping this open as this enhancement will (eventually) be implemented, but don't hold your breath as it won't be anytime soon.

**Comment by jacksongoode** [comment]
> I'm keeping this open as this enhancement will (eventually) be implemented, but don't hold your breath as it won't be anytime soon.

Of course! I imagine this would be a lot of work - though hopefully you can find a ready-made bridge to get you there quickly (and iterate afterwards)!

Perhaps in the mean time, being able to _see_ who is in a chat room (even without being able to chat) would be a really nice stepping stone on the way there.

**Comment by OpenSource03** [comment]
I would very much prioritise this feature if I was you, as this is the most important one really... I'm not on Discord to write chat messages or add reactions to those. 90% of the time on Discord I (and basically everybody else) spend on calls. This should be the number 1 priority...

**Comment by cryptoAlgorithm** [comment]
@OpenSource03 Understand, except this feature is also the hardest and lengthiest to implement, which means I need to pause other WIP improvements like performance fixes which I'm in the midst of. 

After hearing how important the voice feature is to you and others, I'll make it a priority to work on after fixing the #i-am-speed branch. I'd like to base it off that branch, but it's currently kinda broken.

**Comment by cryptoAlgorithm** [comment]
As always, I'd definitely love contributions :)
